### PR TITLE
Rating component done

### PR DESCRIPTION
**Pull request by TJain02**
Screenshot (129)

### CODE REVIEW

**Comment by sohan9819** [comment]
Looks good to me . Pls host it on netlify so that we can see a live preview of this .

**Review by sohan9819 — APPROVED** [review]
looks good to merge

**Comment by TJain02** [comment]
> looks good to merge

Will host the complete component library itself.

**Review by shubhmehra — APPROVED** [review]
Done approved

File: CSS_Components/rating.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import '/CSS_Components/card.css';
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap');
```
Comment (shubhmehra) [inline CSS_Components/rating.css:2]: why you used this fonts import multiple time..
you can import that in main file right!

**Review by shubhmehra — APPROVED** [review]

File: CSS_Components/rating.css (hunk 1 above)
Comment (TJain02) [reply CSS_Components/rating.css:2]: > why you used this fonts import multiple time.. you can import that in main file right!

Yes yes will do that.

**Pull request merged**